assets/js/wfparameters/ColumnSelector.js

```diff
@@ -62,7 +66,11 @@ export default class ColumnSelector extends React.PureComponent {
         value: column
       }
     ))
-
+    const maybeButton = initialValue === value ? null : (
+      <button title="submit" ref={this.buttonRef} onClick={this.props.onSubmit}>
+        <i className="icon-play" />
+      </button>
+    )
     return (
       // The name attributes in the buttons are used for selection in tests. Do not change them.
       <div>
```

Comment:
We should put a `className="column-selector"` here (and that's why we shouldn't put `column-multi-select` later... read on....)

(Why not "column-multi-select"? Consistency. Stick with the same name in CSS and JS. If it irks you enough, please pull-request a rename of the entire component :) -- but not in _this_ pull request)